src/components/icon/icon.js

```diff
@@ -101,7 +101,6 @@ const Icon = TooltipDecorator(class Icon extends React.Component {
     delete props.bgSize;
     delete props.bgShape;
     delete props.bgTheme;
-    props.type = this.type;
```

Comment:
Would there be any use in having this as a data attribute?

Reply:
Maybe. Although it's already included in the `className` e.g. `icon-tick`.